typescript/approvals.d.ts

```diff
@@ -206,7 +206,12 @@ export type FormSubmissionApproval = BaseFormSubmissionApproval & {
   /** The unique identifier for the record */
   id: string
   /** The status of the approval */
-  status: 'PENDING' | 'APPROVED' | 'CLARIFICATION_REQUIRED' | 'CLOSED'
+  status:
+    | 'PENDING'
+    | 'APPROVED'
+    | 'CLARIFICATION_REQUIRED'
+    | 'CLOSED'
+    | 'CANCELLED'
```

Comment:
I don't think `CANCELLED` is a suitable verb for this.

Reply:
Noted.